### PR TITLE
create inpute file tool.

### PR DESCRIPTION
**Pull request by Smeds**
### This PR:

feat: new command line option to generate samples.tsv and units.tsv files.

Needs unit test.

### Review and tests: 
- [x] Tests pass
- [x] Code review

### CODE REVIEW

**Review by jonca79 — APPROVED** [review]

File: hydra_genetics/__main__.py [hunk 1]
```diff
@@ -127,16 +127,110 @@ def create_module(name, description, author, email, version, min_snakemake_versi
...
+@click.option(
+        "--tc",
+        help="tumor contet",
+        type=float,
+        default=1.0)
+def create_input_files(directory, outdir, post_file_modifier, platform, run, sample_type,
+                       sample_regex, read_number_regex, lane_identifier, adapters, tc, force):
+    input_files = CreateInputFiles(directory, outdir, post_file_modifier, platform, run, sample_type,
+                                   sample_regex, read_number_regex, lane_identifier, adapters, tc, force)
+    input_files.init_rule()
```
Comment (Smeds) [inline hydra_genetics/__main__.py:231]:
```suggestion
    input_files.init()
```

File: hydra_genetics/commands/create.py [hunk 2]
```diff
@@ -204,3 +206,137 @@ def init_rule(self):
...
+        self.read_number_regex = read_number_regex
+        self.lane_identifier = lane_identifier
+        self.adapters = adapters
+        self.tc = tc
+        self.force = force
+
+        if not self.outdir:
+            self.outdir = os.getcwd()
+
+    def init_rule(self):
```
Comment (Smeds) [inline hydra_genetics/commands/create.py:246]:
```suggestion
    def init(self):
```

File: hydra_genetics/__main__.py (hunk 1 above)
Comment (Smeds) [inline hydra_genetics/__main__.py:231]:
```suggestion
    input_files.init()
```

File: hydra_genetics/commands/create.py [hunk 3]
```diff
@@ -243,7 +243,7 @@ def __init__(self,
         if not self.outdir:
             self.outdir = os.getcwd()
 
-    def init_rule(self):
+    def init(self):
         """Creates the hydra_genetics rule."""
```
Comment (marrip) [inline hydra_genetics/commands/create.py:247]: another minor thing, here it says rule 😉

**Pull request merged**